<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="dStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(255, 0, 0);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(251, 203, 67);
background-color: rgb(79, 102, 102);</string>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>800</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(211, 219, 43);
background-color: rgb(27, 27, 27);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(230, 127, 118);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 27);
color: rgb(230, 127, 118);</string>
   </property>
   <property name="text">
    <string>温湿度监控报警系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="eTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="eStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(0, 255, 0);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(255, 206, 68);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>C区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>E区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="menuPushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 140, 245);
font: 32pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 140, 245);
font: 32pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>B区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 140, 245);
font: 32pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>F区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(26, 162, 96);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fStateLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
background-color: rgb(255, 255, 0);
color: rgb(124, 177, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(251, 203, 67);
background-color: rgb(79, 102, 102);</string>
   </property>
   <property name="text">
    <string>温度/℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="eHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aHumidityLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>D区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(251, 203, 67);
background-color: rgb(79, 102, 102);</string>
   </property>
   <property name="text">
    <string>湿度/%RH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Ubuntu&quot;;
color: rgb(251, 203, 67);
background-color: rgb(79, 102, 102);</string>
   </property>
   <property name="text">
    <string>区域</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(23, 106, 171);</string>
   </property>
   <property name="text">
    <string>A区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cTemperatureLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);
font: 20pt &quot;Ubuntu&quot;;
color: rgb(113, 170, 61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>quitPushButton</zorder>
  <zorder>dStateLabel</zorder>
  <zorder>label_31</zorder>
  <zorder>label_3</zorder>
  <zorder>fTemperatureLabel</zorder>
  <zorder>bHumidityLabel</zorder>
  <zorder>cStateLabel</zorder>
  <zorder>label</zorder>
  <zorder>eTemperatureLabel</zorder>
  <zorder>eStateLabel</zorder>
  <zorder>bStateLabel</zorder>
  <zorder>cHumidityLabel</zorder>
  <zorder>label_10</zorder>
  <zorder>label_16</zorder>
  <zorder>dTemperatureLabel</zorder>
  <zorder>fHumidityLabel</zorder>
  <zorder>menuPushButton</zorder>
  <zorder>resetPushButton</zorder>
  <zorder>label_9</zorder>
  <zorder>label_17</zorder>
  <zorder>aTemperatureLabel</zorder>
  <zorder>aStateLabel</zorder>
  <zorder>fStateLabel</zorder>
  <zorder>label_6</zorder>
  <zorder>eHumidityLabel</zorder>
  <zorder>dHumidityLabel</zorder>
  <zorder>aHumidityLabel</zorder>
  <zorder>label_15</zorder>
  <zorder>label_11</zorder>
  <zorder>label_5</zorder>
  <zorder>label_8</zorder>
  <zorder>bTemperatureLabel</zorder>
  <zorder>cTemperatureLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
